yarn/src/main/scala/org/apache/spark/deploy/yarn/ExecutorRunnable.scala

```diff
@@ -146,11 +146,11 @@ class ExecutorRunnable(
     javaOpts += "-Xmx" + executorMemoryString
 
     // Set extra Java options for the executor, if defined
-    sys.props.get("spark.executor.extraJavaOptions").foreach { opts =>
-      javaOpts ++= Utils.splitCommandString(opts).map(YarnSparkHadoopUtil.escapeForShell)
-    }
+    sys.props.get("spark.executor.extraJavaOptions").map(
```

Comment:
Sorry, I was looking for something like "SelectMany" from .Net and after not finding it, I forgot to turn it back. Very sloppy of me, my apologies....